erroranalysis/erroranalysis/_internal/matrix_filter.py

```diff
@@ -268,13 +306,59 @@ def _agg_func_triplet(self, pair):
         (true_y, pred_y) = zip(*pair.values.tolist())
         return self.aggfunc(true_y, pred_y)
 
+    def _fill_na_value(self):
+        return 0
+
+
+class _MultiMetricAggFunc(_BaseAggFunc):
+    def __init__(self, aggfunc, metric, labels):
+        super(_MultiMetricAggFunc, self).__init__(aggfunc)
+        self.metric = metric
+        if len(labels) == 2:
+            # for binary classification case, choose positive class label
+            self.labels = [labels[1]]
```

Comment:
In the future we may want to make it possible to select the positive class. This is fine for now